hugo/themes/piercer/layouts/partials/head.html

```diff
@@ -15,12 +15,11 @@
 
 	{{ template "_internal/twitter_cards.html" . }}
 	{{ template "_internal/opengraph.html" . }}
-    <!-- quick and dirty, do this properly! -->
-  <meta property="og:image" content="https://klimawatch.codefor.de/img/klimawatch_gruen.png" />
-		<meta name="twitter:card" content="summary_large_image"/>
-<meta name="twitter:image" content="https://klimawatch.codefor.de/img/klimawatch_gruen.png"/>
-
-  <!-- end quick and dirty -->
+	<!-- quick and dirty, do this properly! -->
+	<meta property="og:image" content="https://klimawatch.codefor.de/img/klimawatch_gruen.png" />
```

Comment:
Diese meta-properties werden übrigens automatisch von den beiden obigen templates (twitter-cards bzw. opengraph) automatisch gesetzt, *wenn* in der jeweiligen Markdown-Datei das image-tag genutzt wird. Ist m.E. die saubere Lösung, dann könnten die drei Zeilen hier weg. Und man könnte für jede Stadt ein anderes Bild einfügen. Beißt sich aber momentan mit dem transparenten hero-background-Bild. Ist aber wohl was für einen anderen PR.

Reply:
Es wäre denkbar `images` wieder einzuführen und einen zusätzlichen Seitenparameter `hero_image` oder so ähnlich für das transparente Bild zu verwenden.

Reply:
Sollten wir im Hinterkopf haben. Wäre ja ganz schön, wenn einzelne Kommunen ein eigenes Bild haben wollen.